### PR TITLE
3 layout

### PR DESCRIPTION
**Pull request by lchristus**

### CODE REVIEW

File: src/MensaWidget/MWFrameTest.class.st [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	widgetFrame := MWFrame new
+]
+
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 17:10'
+}
+MWFrameTest >> tearDown [
+
+	widgetFrame abandon
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWFrameTest.class.st:36]: if we agree on making accessors, we should use them here

File: src/MensaWidget/MWFrame.class.st [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 17:09'
+}
+MWFrame >> addText [
+
+	| textMorph |
+	
+	textMorph := TextMorph new.
+		textMorph contents: 'Are you a Munch? ;)'.
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWFrame.class.st:23]: why is this indented?

File: src/MensaWidget/MWFrame.class.st [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		textMorph position:  135@100.
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWFrame.class.st:24]: you can use a cascade here?

File: src/MensaWidget/MWFrame.class.st [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWFrame.class.st:25]: Do we want to agree on doing breaklines before self?

**Review by Kangaroo6942 — COMMENTED** [review]
Only minor styling questions - functionality as expected

File: src/MensaWidget/MWFrame.class.st (hunk 4 above)
Comment (lchristus) [reply src/MensaWidget/MWFrame.class.st:25]: no i mistakenly did

File: src/MensaWidget/MWFrameTest.class.st (hunk 1 above)
Comment (lchristus) [reply src/MensaWidget/MWFrameTest.class.st:36]: done

File: src/MensaWidget/MWFrame.class.st (hunk 2 above)
Comment (lchristus) [reply src/MensaWidget/MWFrame.class.st]: done

File: src/MensaWidget/MWFrame.class.st (hunk 3 above)
Comment (lchristus) [reply src/MensaWidget/MWFrame.class.st]: done

File: src/MensaWidget/MWFrame.class.st (hunk 4 above)
Comment (lchristus) [reply src/MensaWidget/MWFrame.class.st]: done

File: src/MensaWidget/MWFrame.class.st [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 5/29/2024 18:43'
+}
+MWFrame >> addText [
+
+	| textMorph |
+	
+	textMorph := TextMorph new.
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWFrame.class.st:22]: currently the text morph doesn't show up in the widget window on my scaling

Comment (Kangaroo6942) [reply src/MensaWidget/MWFrame.class.st:22]: aye

**Review by Kangaroo6942 — APPROVED** [review]
all resolved

**Pull request merged**